xrpl4j-core/src/main/java/org/xrpl/xrpl4j/model/transactions/Wrappers.java

```diff
@@ -352,12 +352,12 @@ public String toString() {
      * @param percent of type {@link BigDecimal}
      * @return {@link TransferFee}
      */
-    static TransferFee ofPercent(BigDecimal percent) {
```

Comment:
Can we add this to the Javadoc?

```
Valid values for this field are between 0 and 50000 inclusive, allowing transfer rates of between 0.00% and 50.00% in increments of 0.001. If this field is provided, the transaction MUST have the [tfTransferable flag](https://xrpl.org/nftokenmint.html#nftokenmint-flags) enabled.
```

Also, add an `@see https://xrpl.org/nftokenmint.html#nftokenmint-flags`